Francis/blink/blink.ino

```diff
@@ -0,0 +1,41 @@
+
+int led = 6;
+//set led pin to pin 6
+int brightness=0;
+int fadeamount=5;
```

Comment:
watch for consistency with your spacing around the equal signs. Also is your comment on line three referring to line 2 or line 4?